Python/Product/Debugger/Debugger/PythonThread.cs

```diff
@@ -31,34 +33,30 @@ class PythonThread {
             _name = "";
         }
 
-        public void StepInto() {
-            _process.SendStepInto(_identity);
+        public Task StepIntoAsync(CancellationToken ct) {
+            return _process.SendStepIntoAsync(_identity, ct);
```

Comment:
I changed it. It would have been better if it fit on line, but that made some of the lines too long.  So in the end we're saving one line per method.

---
In reply to: [107987565](https://github.com/Microsoft/PTVS/pull/2323#discussion_r107987565) [](ancestors = 107987565)